src/nvim/drawline.c

```diff
@@ -427,19 +429,11 @@ void fill_foldcolumn(void *maybe_wlv, win_T *wp, foldinfo_T foldinfo, linenr_T l
     if (out_buffer) {
       out_buffer[i] = schar_from_char(symbol);
     } else {
-      linebuf_vcol[wlv->off] = (i == endcol - 1 && closed) ? -2 : -3;
```

Comment:
A better simpler change which wouldn't break the semantics of this function would be to just change the first parameter as `int *wlv_off` and then just use that.